### PR TITLE
Print more details of SSL certificates

### PR DESCRIPTION
**Pull request by mxj4**
Print more details of SSL certificates

Some server program requires path / fingerprints to SSL certificates.
Having these information in the UI simplifies the setup process.

Fixes: [ISSUE_URL or #ISSUE_ID, create one if necessary]

Signed-off-by: [YOUR_NAME] <[YOUR_EMAIL]>

- [ ] References issue
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by votdev** [comment]
- Please add a screenshot if possible that shows what has been improved.
- Please squash your commits. Only one commit per PR is required.
- Please add the `Signed-off-by: [YOUR_NAME] <[YOUR_EMAIL]>` line to the commit message, otherwise the PR can not be reviewed and merged.

File: deb/openmediavault/usr/share/openmediavault/engined/rpc/certificatemgmt.inc [hunk 1]
```diff
@@ -218,19 +218,46 @@ class CertificateMgmt extends \OMV\Rpc\ServiceAbstract {
...
+		);
+		$output[] = sprintf("Private Key Path: %s", $keyPath);
+		// Certificate path.
+		$certPath = sprintf(
+			"%s/certs/%s%s.crt",
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_DIR"),
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_PREFIX"),
+			$params['uuid'],
+		);
+		$output[] = sprintf("Certificate Path: %s", $certPath);
```
Comment (votdev) [inline deb/openmediavault/usr/share/openmediavault/engined/rpc/certificatemgmt.inc:237]: I do not like to show this information in the UI. This is against the philosophy of OMV which is to hide sensitive data in the UI. Please remove that.

File: deb/openmediavault/usr/share/openmediavault/engined/rpc/certificatemgmt.inc [hunk 2]
```diff
@@ -218,19 +218,46 @@ class CertificateMgmt extends \OMV\Rpc\ServiceAbstract {
...
 			throw new \OMV\Config\ConfigDirtyException();
+		$output = [];
+		// Private key path.
+		$keyPath = sprintf(
+			"%s/private/%s%s.key",
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_DIR"),
+			\OMV\Environment::get("OMV_SSL_CERTIFICATE_PREFIX"),
+			$params['uuid'],
+		);
+		$output[] = sprintf("Private Key Path: %s", $keyPath);
```
Comment (votdev) [inline deb/openmediavault/usr/share/openmediavault/engined/rpc/certificatemgmt.inc:229]: I do not like to show this information in the UI. This is against the philosophy of OMV which is to hide sensitive data in the UI. Please remove that.

**Review by votdev — CHANGES_REQUESTED** [review]

**Comment by votdev** [comment]
> Some server program requires path / fingerprints to SSL certificates.

Showing the fingerprints is OK, but the paths are against the philosophy of OMV. A plugin should handle that for the user under the hood by only showing the available SSL certs in a dropdown box.

**Comment by mxj4** [comment]
> > Some server program requires path / fingerprints to SSL certificates.
> 
> Showing the fingerprints is OK, but the paths are against the philosophy of OMV. A plugin should handle that for the user under the hood by only showing the available SSL certs in a dropdown box.

thx for the review. Can I show the UUID of the certificate? Otherwise users need to search the config XML to find the UUID then construct the full path.

**Comment by votdev** [comment]
> > > Some server program requires path / fingerprints to SSL certificates.
> > 
> > 
> > Showing the fingerprints is OK, but the paths are against the philosophy of OMV. A plugin should handle that for the user under the hood by only showing the available SSL certs in a dropdown box.
> 
> thx for the review. Can I show the UUID of the certificate? Otherwise users need to search the config XML to find the UUID then construct the full path.

What is the exact usecase for this feature? I have a bad feeling in my stomach about publishing this information.

**Pull request closed without merging**